### PR TITLE
Add warning message in pdf 

### PR DESCRIPTION
**Pull request by oterral**
Display third data party copyright in red in the pdf and add a warning message

### CODE REVIEW

**Comment by oterral** [comment]
to merge after the deploy of 15/04

File: print/config.yaml [hunk 1]
```diff
@@ -95,74 +95,110 @@ layouts:
...
-                www.geo.admin.ch è in portal d'access ad infurmaziuns geolocalisadas, a datas ed a servetschs che vegnan mess a disposiziun da l'administraziun federala
+                Warnung: diese Karten enthält Daten von Drittanbietern
             - !text
               condition: langit
+              fontColor: red
               fontSize: 7
               align: left
               text: |
-                www.geo.admin.ch è una piattaforma pubblica accessibile per la ricerca di geo-informazioni, i dati e servizi.
+                Attenzione: questi dati provengono da terze parti.
```
Comment (AFoletti) [inline print/config.yaml]: Better IT:
Attenzione: alcuni dati provengono da terze parti

File: print/config.yaml [hunk 2]
```diff
@@ -95,74 +95,110 @@ layouts:
...
-                www.geo.admin.ch ist ein Portal zur Einsicht von geolokalisierten Informationen, Daten und Diensten, die von öffentlichen Einrichtungen zur Verfügung gestellt werden
+                Warnung: diese Karten enthält Daten von Drittanbietern
             - !text
               condition: langfr
+              fontColor: red
               fontSize: 7
               align: left
               text: |
-                www.geo.admin.ch est un portail d'accès aux informations géolocalisées, données et services qui sont mis à disposition par l'administration fédérale
+                Attention: cette carte contient des donné tierces.
```
Comment (procrastinatio) [inline print/config.yaml]: Hem,
`Attention: cette carte contient des données tierces.`

**Comment by procrastinatio** [comment]
Looks good. Your example breaks the 'multiprint', but here is the fix: https://github.com/geoadmin/mf-chsdi3/pull/1373

**Comment by procrastinatio** [comment]
jenkins test

**Comment by oterral** [comment]
Ready to review/merge after today's deploy

**Comment by oterral** [comment]
Thx

**Comment by cedricmoullet** [comment]
We need to wait the go from @davidoesch that should come tomorrow

**Pull request merged**